src/test/coins_tests.cpp

```diff
@@ -51,25 +51,19 @@ class CCoinsViewTest : public CCoinsView
             return false;
         }
         coin = it->second;
-        if (coin.IsSpent() && m_rng.randbool() == 0) {
```

Comment:
Did this change require any other test adjustment?

Reply:
No, but without this the next commit would break.